app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -286,7 +284,7 @@ div.picture_gallery_images {
   }
 }
 
-div.picture_thumbnail div.picture_image {
+.picture_thumbnail .picture_image {
```

Comment:
Selector `picture_thumbnail` should be written in lowercase with hyphens<br>Selector `picture_image` should be written in lowercase with hyphens